testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -739,5 +739,5 @@ def test_atomname_alignment(self):
 
     def test_atomtype_alignment(self):
         result_line = ("ATOM      1  H5T GUA R   1       7.974   6.430   9.561"
-                       "  1.00  0.00      RNAA H0.00\n")
-        assert_equal(result_line, self.writtenstuff[4])
+                       "  1.00  0.00      RNAA H 0\n")
+        assert_equal(self.writtenstuff[4], result_line)
```

Comment:
A test with a negative charge would be nice. But it should go in an other test method, or the name of this test method should be changed since its scope would be different.